ansible_collections/arista/avd/molecule/eos_designs_unit_tests/intended/configs/cv-pathfinder-edge-wan-use-evpn-on-lan-no-overlay-on-lan.cfg

```diff
@@ -0,0 +1,372 @@
+      avt profile PROD-AVT-POLICY-VIDEO id 4
+!
+router path-selection
+   tcp mss ceiling ipv4 ingress
+   !
+   path-group INET id 101
+      ipsec profile CP-PROFILE
+      !
+      local interface Ethernet1
+         stun server-profile INET-cv-pathfinder-pathfinder-Ethernet1 INET-cv-pathfinder-pathfinder-Ethernet3
+      !
+      peer dynamic
+      !
+      peer static router-ip 192.168.144.1
+         name cv-pathfinder-pathfinder
+         ipv4 address 172.17.7.7
+         ipv4 address 10.9.9.9
+   !
+   load-balance policy LB-DEFAULT-AVT-POLICY-CONTROL-PLANE
+      path-group INET
+   !
+   load-balance policy LB-DEFAULT-AVT-POLICY-DEFAULT
+      path-group INET
+   !
+   load-balance policy LB-DEFAULT-AVT-POLICY-VIDEO
+      path-group INET
+   !
+   load-balance policy LB-PROD-AVT-POLICY-DEFAULT
+      path-group INET
+   !
+   load-balance policy LB-PROD-AVT-POLICY-VIDEO
+      loss-rate 42.0
+      path-group INET priority 2
+   !
+   load-balance policy LB-PROD-AVT-POLICY-VOICE
+      jitter 42
+      hop count lowest
+      path-group INET priority 2
+!
+spanning-tree mode none
+!
+vrf instance MGMT
+!
+vrf instance PROD
+!
+vrf instance VRF-NO-WAN
+!
+vrf instance VRF-NO-WAN-NO-AF
+!
+vrf instance WAN-VRF-NO-AF
+!
+management api http-commands
+   protocol https
+   no shutdown
+   !
+   vrf MGMT
+      no shutdown
+!
+management security
+   !
+   ssl profile profileA
+      tls versions 1.2
+      trust certificate aristaDeviceCertProvisionerDefaultRootCA.crt
+      certificate profileA.crt key profileA.key
+!
+ip security
+   ike policy CP-IKE-POLICY
+      local-id 192.168.142.14
+   !
+   sa policy CP-SA-POLICY
+      esp encryption aes256gcm128
+      pfs dh-group 14
+   !
+   sa policy DP-SA-POLICY
+      esp encryption aes256gcm128
+      pfs dh-group 14
+   !
+   profile CP-PROFILE
+      ike-policy CP-IKE-POLICY
+      sa-policy CP-SA-POLICY
+      connection start
+      shared-key 7 ABCDEF1234567890
+      dpd 10 50 clear
+      mode transport
+   !
+   profile DP-PROFILE
+      sa-policy DP-SA-POLICY
+      connection start
+      shared-key 7 ABCDEF1234567890666
+      dpd 10 50 clear
+      mode transport
+   !
+   key controller
+      profile DP-PROFILE
+!
+interface Dps1
+   description DPS Interface
+   mtu 9194
+   flow tracker hardware FLOW-TRACKER
+   ip address 192.168.142.14/32
+!
+interface Ethernet1
+   description ATT_666
+   no shutdown
+   no switchport
+   ip address dhcp
+   dhcp client accept default-route
+!
+interface Ethernet52
+   description P2P_leaf-wan-use-evpn-on-lan_Ethernet2
+   no shutdown
+   mtu 9214
+   no switchport
+   ip address 172.18.0.27/31
+!
+interface Loopback0
+   description ROUTER_ID
+   no shutdown
+   ip address 192.168.42.14/32
+!
+interface Vxlan1
+   description cv-pathfinder-edge-wan-use-evpn-on-lan-no-overlay-on-lan_VTEP
+   vxlan source-interface Dps1
+   vxlan udp-port 4789
+   vxlan vrf default vni 1
+   vxlan vrf PROD vni 42
+   vxlan vrf VRF-NO-WAN vni 300
```

Comment:
@ClausHolbechArista right now this is stil generated and I am not sure we want that.

Reply:
no we don't